### PR TITLE
Fix connection flow control

### PR DESCRIPTION
**Pull request by anisimkov**
Do not need update connection flow control together with stream
flow control. Connection flow control has to be updated separately.
Fixes file dowmload longer than 2G.

TN: S507-051

### CODE REVIEW

File: src/http2/aws-http2-frame.ads [hunk 1]
```diff
@@ -222,6 +222,8 @@ private
    overriding procedure Adjust (Self : in out Object);
    overriding procedure Finalize (Self : in out Object);
 
+   procedure Dump_Binary (Data : Stream_Element_Array);
```
Comment (TurboGit) [inline src/http2/aws-http2-frame.ads:225]: to be moved in AWS.Utils as generic.

File: src/core/aws-client-http_utils.adb [hunk 2]
```diff
@@ -417,6 +424,41 @@ package body AWS.Client.HTTP_Utils is
...
+   -- Get_H2_Frame --
+   ------------------
+
+   procedure Get_H2_Frame
+     (Connection : in out HTTP_Connection;
+      Ctx        : in out Server.Context.Object;
+      Stream     : in out HTTP2.Stream.Object)
+   is
+      use AWS.HTTP2;
+      Frame : constant HTTP2.Frame.Object'Class :=
```
Comment (TurboGit) [inline src/core/aws-client-http_utils.adb:437]: indent :

**Review by TurboGit — CHANGES_REQUESTED** [review]
Just two minor comments, look ok otherwise!

File: src/core/aws-client-http_utils.adb (hunk 2 above)
Comment (anisimkov) [reply src/core/aws-client-http_utils.adb]: Moved

File: src/http2/aws-http2-frame.ads (hunk 1 above)
Comment (anisimkov) [reply src/http2/aws-http2-frame.ads:225]: Moved

**Pull request merged**